explainer.md

```diff
@@ -1,3 +1,118 @@
 ## Explainer
```

Comment:
Detailed and thorough document, very nice!

Can you just add a paragraph introducing the document as a whole. The spec is doing multiple different things, but this explainer makes it sound like it is all about capture timestamp.

I don't mean to write as thoroughly about the other stuff, I mean just an introduction. Basically what the [introduction section](https://w3c.github.io/webrtc-extensions/#introduction) is already saying.